modules/waf/main.tf

```diff
@@ -0,0 +1,10 @@
+##############################################################################
+# To enable/disable Web Application Firewall(WAF) for a domain
+##############################################################################
+
+resource "ibm_cis_domain_settings" "domain_settings" {
+  cis_id          = var.cis_instance_id
+  domain_id       = var.domain_id
+  waf             = var.waf
+  min_tls_version = var.min_tls_version
```

Comment:
We discussed `not to use` as variable...because we are not asking user to choose tls_version. We just want to keep it as default.

Reply:
Also, a valid comment is required here....why are we doing this

Reply:
Done